src/main/resources/db/migration/V20200501105700__Create_shared_recipe_list_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE shared_recipe_list(
+    id SERIAL PRIMARY KEY,
+    created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    recipe_list_id BIGINT NOT NULL REFERENCES recipe_list(id),
```

Comment:
Denne kunne vel være null?

Reply:
Hmm, gir det mening å dele en liste som ikke finnes?

Reply:
Mente denne:
recipient_id

Reply:
Ah, ja. Den har ikke `NOT NULL` constraint, så den skal kunne være `NULL`.